functional tests raising `pydantic.errors.ConfigError: Invalid signature for root validator`

see https://github.com/onicagroup/runway/actions/runs/10058710452/job/27802451166#step:11:183

```text
_ ERROR collecting tests/functional/cfngin/hooks/test_awslambda/test_runner.py _
tests/functional/cfngin/hooks/test_awslambda/test_runner.py:54: in <module>
    class AwslambdaStackOutputs(BaseModel):
pydantic/main.py:246: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/class_validators.py:220: in pydantic.class_validators.extract_root_validators
    ???
E   pydantic.errors.ConfigError: Invalid signature for root validator _convert_null_to_none: (self, values: 'dict[str, Any]') -> 'dict[str, Any]', "self" not permitted as first argument, should be: (cls, values).
_ ERROR collecting tests/functional/cfngin/hooks/test_awslambda/test_runner.py _
tests/functional/cfngin/hooks/test_awslambda/test_runner.py:54: in <module>
    class AwslambdaStackOutputs(BaseModel):
pydantic/main.py:246: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/class_validators.py:220: in pydantic.class_validators.extract_root_validators
    ???
E   pydantic.errors.ConfigError: Invalid signature for root validator _convert_null_to_none: (self, values: 'dict[str, Any]') -> 'dict[str, Any]', "self" not permitted as first argument, should be: (cls, values).
_ ERROR collecting tests/functional/cfngin/hooks/test_awslambda/test_runner.py _
tests/functional/cfngin/hooks/test_awslambda/test_runner.py:54: in <module>
    class AwslambdaStackOutputs(BaseModel):
pydantic/main.py:246: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/class_validators.py:220: in pydantic.class_validators.extract_root_validators
    ???
E   pydantic.errors.ConfigError: Invalid signature for root validator _convert_null_to_none: (self, values: 'dict[str, Any]') -> 'dict[str, Any]', "self" not permitted as first argument, should be: (cls, values).
_ ERROR collecting tests/functional/cfngin/hooks/test_awslambda/test_runner.py _
tests/functional/cfngin/hooks/test_awslambda/test_runner.py:54: in <module>
    class AwslambdaStackOutputs(BaseModel):
pydantic/main.py:246: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/class_validators.py:220: in pydantic.class_validators.extract_root_validators
    ???
E   pydantic.errors.ConfigError: Invalid signature for root validator _convert_null_to_none: (self, values: 'dict[str, Any]') -> 'dict[str, Any]', "self" not permitted as first argument, should be: (cls, values).
```

--- a/tests/functional/cfngin/hooks/test_awslambda/test_runner.py
+++ b/tests/functional/cfngin/hooks/test_awslambda/test_runner.py
@@ -69,7 +69,7 @@ class AwslambdaStackOutputs(BaseModel):
     Runtime: str
 
     @root_validator(allow_reuse=True, pre=True)
-    def _convert_null_to_none(self, values: dict[str, Any]) -> dict[str, Any]:
+    def _convert_null_to_none(cls, values: dict[str, Any]) -> dict[str, Any]:  # noqa: N805
         """Convert ``null`` to ``NoneType``."""
 
         def _handle_null(v: Any) -> Any:

